src/components/Laout.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import { NavLink, Outlet } from 'react-router-dom';
+
+const Laout = () => {
```

Comment:
The component name `Laout` seems to be a typo. It should likely be `Layout`. Update the component name to ensure consistency and correct functionality.